types/accesscontrol/resource.go

```diff
@@ -54,7 +54,46 @@ var ResourceTree = map[ResourceType]TreeNode{
 	ResourceType_KV_ORACLE_VOTE_TARGETS:    			{ResourceType_KV_ORACLE, []ResourceType{}},
 	ResourceType_KV_ORACLE_AGGREGATE_VOTES: 			{ResourceType_KV_ORACLE, []ResourceType{}},
 	ResourceType_KV_ORACLE_FEEDERS:         			{ResourceType_KV_ORACLE, []ResourceType{}},
-	ResourceType_KV_DEX:                    			{ResourceType_KV, []ResourceType{}},
+	ResourceType_KV_DEX:                    			{ResourceType_KV, []ResourceType{
+		ResourceType_KV_DEX_CONTRACT_LONGBOOK,
+		ResourceType_KV_DEX_CONTRACT_SHORTBOOK,
+		ResourceType_KV_DEX_ORDER_BOOK,
+		ResourceType_KV_DEX_PAIR_PREFIX,
+		ResourceType_KV_DEX_TWAP,
+		ResourceType_KV_DEX_PRICE,
+		ResourceType_KV_DEX_SETTLEMENT_ENTRY,
+		ResourceType_KV_DEX_REGISTERED_PAIR,
+		ResourceType_KV_DEX_TICK_SIZE,
+		ResourceType_KV_DEX_ORDER,
+		ResourceType_KV_DEX_CANCEL,
+		ResourceType_KV_DEX_ACCOUNT_ACTIVE_ORDERS,
+		ResourceType_KV_DEX_REGISTERED_PAIR_COUNT,
+		ResourceType_KV_DEX_ASSET_LIST,
+		ResourceType_KV_DEX_NEXT_ORDER_ID,
+		ResourceType_KV_DEX_NEXT_SETTLEMENT_ID,
+		ResourceType_KV_DEX_MATCH_RESULT,
+		ResourceType_KV_DEX_SETTLEMENT_ORDER_ID,
+		ResourceType_KV_DEX_SETTLEMENT,
+	}},
+	ResourceType_KV_DEX_CONTRACT_LONGBOOK:  			{ResourceType_KV_DEX, []ResourceType{}},
```

Comment:
Weird formatting on git, it looks fine locally
![image](https://user-images.githubusercontent.com/18161326/199272266-0acc2380-5f08-4cea-81f1-5410259281d9.png)